codegen/server/example_cli.go

```diff
@@ -0,0 +1,222 @@
+package server
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"goa.design/goa/codegen"
+	"goa.design/goa/expr"
+)
+
+// ExampleCLIFiles returns example client tool main implementation for each
+// server expression in the design.
```

Comment:
Can you remind me why we decided to generate multiple main - one per server - vs. having multiple command line flags? might be good to put that explanation in the comments too.

Reply:
Wasn't that your original intention behind using the server expression in generating example mains? One for each server expression. The hosts and URI variables within each server expression will get their own CLI args.

Reply:
Duh yeah you are right. One server generated per server expression :)